Can't compile in kali amd64

I tried the dkms compile and the other one, both ones produce the same error.
Here is output of sudo make all: `


  CC [M]  /home/TL-WN722N-V2/os_dep/linux/os_intfs.o
In file included from /home/TL-WN722N-V2/include/drv_types.h:35,
                 from /home/TL-WN722N-V2/os_dep/linux/os_intfs.c:22:
/home/TL-WN722N-V2/include/wifi.h:988: warning: "IEEE80211_MAX_AMPDU_BUF" redefined
 #define IEEE80211_MAX_AMPDU_BUF 0x40
 
In file included from /usr/src/linux-headers-4.19.0-kali1-common/include/net/cfg80211.h:24,
                 from /home/TL-WN722N-V2/include/osdep_service_linux.h:83,
                 from /home/TL-WN722N-V2/include/osdep_service.h:41,
                 from /home/TL-WN722N-V2/include/drv_types.h:32,
                 from /home/TL-WN722N-V2/os_dep/linux/os_intfs.c:22:
/usr/src/linux-headers-4.19.0-kali1-common/include/linux/ieee80211.h:1442: note: this is the location of the previous definition
 #define IEEE80211_MAX_AMPDU_BUF  0x100
 
/home/TL-WN722N-V2/os_dep/linux/os_intfs.c:751:22: error: initialization of ‘u16 (*)(struct net_device *, struct sk_buff *, struct net_device *, u16 (*)(struct net_device *, struct sk_buff *, struct net_device *))’ {aka ‘short unsigned int (*)(struct net_device *, struct sk_buff *, struct net_device *, short unsigned int (*)(struct net_device *, struct sk_buff *, struct net_device *))’} from incompatible pointer type ‘u16 (*)(struct net_device *, struct sk_buff *, void *, u16 (*)(struct net_device *, struct sk_buff *, struct net_device *))’ {aka ‘short unsigned int (*)(struct net_device *, struct sk_buff *, void *, short unsigned int (*)(struct net_device *, struct sk_buff *, struct net_device *))’} [-Werror=incompatible-pointer-types]
  .ndo_select_queue = rtw_select_queue,
                      ^~~~~~~~~~~~~~~~
/home/TL-WN722N-V2/os_dep/linux/os_intfs.c:751:22: note: (near initialization for ‘rtw_netdev_ops.ndo_select_queue’)
cc1: some warnings being treated as errors
make[4]: *** [/usr/src/linux-headers-4.19.0-kali1-common/scripts/Makefile.build:309: /home/TL-WN722N-V2/os_dep/linux/os_intfs.o] Error 1
make[3]: *** [/usr/src/linux-headers-4.19.0-kali1-common/Makefile:1532: _module_/home/TL-WN722N-V2] Error 2
make[2]: *** [Makefile:146: sub-make] Error 2
make[1]: *** [Makefile:8: all] Error 2
make[1]: Leaving directory '/usr/src/linux-headers-4.19.0-kali1-amd64'
make: *** [Makefile:1365: modules] Error 2
`

--- a/os_dep/linux/os_intfs.c
+++ b/os_dep/linux/os_intfs.c
@@ -630,7 +630,11 @@ unsigned int rtw_classify8021d(struct sk_buff *skb)
 
 static u16 rtw_select_queue(struct net_device *dev, struct sk_buff *skb
 #if LINUX_VERSION_CODE >= KERNEL_VERSION(3, 13, 0)
-	, void *accel_priv
+	#if LINUX_VERSION_CODE < KERNEL_VERSION(4, 19, 0)
+		, void *accel_priv
+	#else
+		, struct net_device *sb_dev
+	#endif
 #endif
 #if LINUX_VERSION_CODE >= KERNEL_VERSION(3, 14, 0)
 	, select_queue_fallback_t fallback

